### PR TITLE
Add `isFlippedCropRect` option

### LINKED ISSUE #65: Support for top/left positions for capture
[Issuehunt badges](https://oss.issuehunt.io/r/wulkano/Aperture/issues/65)

Top/left support would be extremely useful in some situations.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $80.00)

- [issuehunt issuehunt](https://oss.issuehunt.io/u/issuehunt) ($80.00)

### Submitted pull Requests
- [#66 Added isFlippedCropRect](https://oss.issuehunt.io/r/wulkano/Aperture/pull/66)
- [#80 Rewrite with ScreenCaptureKit](https://oss.issuehunt.io/r/wulkano/Aperture/pull/80)
---

### Tips

- Checkout the [Issuehunt explorer](https://oss.issuehunt.io/r/wulkano/Aperture/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://oss.issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

### PR DESCRIPTION
**Pull request by JakeMakesStuff**
Fixes #65

### CODE REVIEW

File: Sources/Aperture/Aperture.swift [hunk 1]
```diff
@@ -40,7 +40,8 @@ public final class Aperture: NSObject {
     highlightClicks: Bool,
     screenId: CGDirectDisplayID = .main,
     audioDevice: AVCaptureDevice? = .default(for: .audio),
-    videoCodec: String? = nil
+    videoCodec: String? = nil,
+    isFlippedCropRect: Bool? = false
```
Comment (sindresorhus) [inline Sources/Aperture/Aperture.swift]: This parameter needs to be documented in a doc comment. It should say what it does and what it's useful for.

File: Sources/Aperture/Aperture.swift [hunk 2]
```diff
@@ -51,6 +52,10 @@ public final class Aperture: NSObject {
 
     if let cropRect = cropRect {
       input.cropRect = cropRect
+      if isFlippedCropRect {
+        var screenHeight = CGDisplayScreenSize(screenId).height
```
Comment (sindresorhus) [inline Sources/Aperture/Aperture.swift]:
```suggestion
        let screenHeight = CGDisplayScreenSize(screenId).height
```

File: Sources/Aperture/Aperture.swift (hunk 1 above)
Comment (sindresorhus) [reply Sources/Aperture/Aperture.swift]: And it's should not be an optional.

**Comment by sindresorhus** [comment]
The code here is not compiling. Please ensure the code can compile and is working properly (that it actually correctly flips the crop rect).

**Comment by sindresorhus** [comment]
I think this needs a test. It can just be to run Aperture with certain options and generate a video file with a descriptive name that can be manually verified.

**Comment by skllcrn** [comment]
Let us know if you want to continue working on this fix @JakeMakesStuff!

**Pull request closed without merging**